### DISCUSSION TITLE
R5.0 Scope items mismatch in Microsite and the sign off provided.

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-Knowlg/Community#62] [General Discussions]
@vinukumar-vs  Please confirm on the 5.0 Knwlg scope which is signed off as the tickets mentioned in Microsite and the tickets mentioned in the sign off template are not matching .

**Tickets added in Microsite for release** 

![image](https://user-images.githubusercontent.com/40622169/188622196-027133b7-85e6-420e-a937-1caf89860cee.png)

![image](https://user-images.githubusercontent.com/40622169/188622400-13098389-9e52-42f4-91fc-0c9197882038.png)

The Below 3 tickets are there in the Microsite but the sign off is not provided for these 3 tickets,
SB-28301
SB-30566
SB-30516
Please update the Microsite accordingly.

The below tickets we have received the sign off but its missing in the microsite.
SB-30378
SB-30573
SB-30560
Please update this as well.

Note: Please note the tickets for which sign off is provided should match with the tickets mentioned in the Microsite.

 @ShailandraOjha @luckynvdu @sharathkashyap @chitranshu-keshav @rhwarrier

**Comment by ShailandraOjha** [comment]
@vinukumar-vs Please confirm the scope. Which one is the correct scope for 5.0

**Reply by chitranshu-keshav** [reply]
@AmiableAnil can you please get back on this

**Reply by ShailandraOjha** [reply]
@vinukumar-vs Please update the microsite and confirm, Diksha will be taking the scope from the microsites hence we need to ensure the microsite is updated with the correct scope.
cc: @chitranshu-keshav

**Reply by anunagaraj** [reply]
@vinukumar-vs The Microsite is still not updated with the correct scope for R5.0.0. Please update the tickets .

@ShailandraOjha  @chitranshu-keshav

**Comment by Priyakashi** [comment]
Updated the microsite accordingly, 
SB-28301 - this is just to mention to the adopter about the Knowlg portal
SB-30566 - this will be verified as a part of 5.1 release
SB-30516 - this ticket can be ignored as it is tagged to ed
SB-30378, SB-30573, SB-30560 - updated in the microsite
@vinukumar-vs @ShailandraOjha @anunagaraj

**Comment by anunagaraj** [comment]
@Priyakashi There is still mismatch in the Microsite and the sign off tickets .The below ticket is still missing in Microsite 
SB-30378
SB-30118 is moved to KN-285
Also if SB-30516 is Tagged to ED then pls remove it from Knwlg release 
Kindly update the same and provide an update here.
@ShailandraOjha @chitranshu-keshav
